### PR TITLE
fix the bug when sort is messed up when switching between dashboard tabs

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

File: src/scenes/Dashboard/components/TopActions/index.js [hunk 1]
```diff
@@ -69,14 +61,18 @@ class TopActions extends Component {
 
 TopActions.propTypes = {
```
Comment (dwalintukan) [inline src/scenes/Dashboard/components/TopActions/index.js]: please declare as `static` and move inside class structure

File: src/scenes/Dashboard/components/TopActions/index.js [hunk 2]
```diff
@@ -69,14 +61,18 @@ class TopActions extends Component {
...
   classes: PropTypes.object.isRequired,
+  sortBy: PropTypes.string,
   sortOrderChanged: PropTypes.func,
 };
 
 TopActions.defaultProps = {
```
Comment (dwalintukan) [inline src/scenes/Dashboard/components/TopActions/index.js]: static and move to class

File: src/scenes/Dashboard/index.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ Dashboard.propTypes = {
 };
 
 Dashboard.defaultProps = {
-  sortBy: undefined,
+  sortBy: SortBy.Ascending,
```
Comment (dwalintukan) [inline src/scenes/Dashboard/index.js]: do you think we should have this default in the `reducer` so we dont have to declare a default for each class that uses it?

Comment (liviavinci) [reply src/scenes/Dashboard/index.js]: need to think about this. cuz we not only need the sort order for request but also for display. let's do a reorg later?

**Pull request merged**